test/cpp/tensorexpr/test_loopnest.cpp

```diff
@@ -2245,7 +2245,7 @@ class LoopOrderHelper : public IRVisitor {
     return ordering.str();
   }
 
-  // NOLINTNEXTLINE(cppcoreguidelines-explicit-virtual-functions,modernize-use-override)
+  // NOLINTNEXTLINE(cppcoreguidelines-explicit--functions,modernize-use-override)
```

Comment:
`cppcoreguidelines-explicit--functions` looks like a typo (and perhaps nolint is no longer needed)
```suggestion
```